installer/conf/logrotate.conf

```diff
@@ -6,3 +6,12 @@
     size 100M
     copytruncate
 }
+
+/var/opt/microsoft/omsagent/log/update_run_progress.log {
```

Comment:
please change file name to urp.log. Since the runbook has been configured and deployed. In order to avoid redeploy runbook, please use the same log file name here.